Lib/idlelib/tree.py

```diff
@@ -199,23 +200,22 @@ def update(self):
 
     def draw(self, x, y):
         # XXX This hard-codes too many geometry constants!
-        dy = 20
         self.x, self.y = x, y
         self.drawicon()
         self.drawtext()
         if self.state != 'expanded':
-            return y + dy
+            return y + self.dy
         # draw children
         if not self.children:
             sublist = self.item._GetSubList()
             if not sublist:
                 # _IsExpandable() was mistaken; that's allowed
-                return y+17
+                return y + self.dy
```

Comment:
Perhaps this should be `y + self.dy - 3 to retain relationship of 20 and 17.  Why I need to know when this is triggered.